src/main/kotlin/no/nav/familie/ef/sak/ekstern/stønadsperiode/EksternStønadsperioderService.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.familie.ef.sak.ekstern.stønadsperiode
+
+import no.nav.familie.ef.sak.ekstern.stønadsperiode.util.ArenaPeriodeUtil
+import no.nav.familie.ef.sak.infotrygd.InternPeriode
+import no.nav.familie.ef.sak.infotrygd.PeriodeService
+import no.nav.familie.kontrakter.felles.Datoperiode
+import no.nav.familie.kontrakter.felles.ef.EksternPeriode
+import no.nav.familie.kontrakter.felles.ef.EksternPeriodeMedBeløp
+import no.nav.familie.kontrakter.felles.ef.EksternePerioderRequest
+import no.nav.familie.kontrakter.felles.ef.EksternePerioderResponse
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+
+@Service
+class EksternStønadsperioderService(
+    private val periodeService: PeriodeService,
+) {
+
+    /**
+     * Brukes av arena: OBS: fom/tom-dato begrenses til DAGENS dato dersom det ikke er satt!
+     */
+    fun hentPerioderForAlleStønader(request: EksternePerioderRequest): EksternePerioderResponse {
+        val perioder = periodeService.hentPerioderFraEfOgInfotrygd(request.personIdent)
+        val perioderFraITogEF = ArenaPeriodeUtil.slåSammenPerioderFraEfOgInfotrygd(request, perioder)
+        return EksternePerioderResponse(perioderFraITogEF)
+    }
+
+    fun hentPerioderForOvergangsstønad(request: EksternePerioderRequest): List<EksternPeriode> {
+        return hentPerioderForOvergangsstønadAvgrensetPeriode(
+            personIdent = request.personIdent,
+            fomDato = request.fomDato,
+            tomDato = request.tomDato,
+        ).map {
+            EksternPeriode(
+                personIdent = request.personIdent,
+                fomDato = it.stønadFom,
+                tomDato = it.stønadTom,
+                datakilde = it.datakilde,
+            )
+        }
+    }
+
+    fun hentPerioderForOvergangsstønadMedBeløp(request: EksternePerioderRequest): List<EksternPeriodeMedBeløp> {
```

Comment:
Kan men eventuellt bruke `hentPerioderForOvergangsstønadMedBeløp` i stedet for `hentPerioderForOvergangsstønad` og? Konsumenten får beløp også, men det kanskje er greit?